k8s/helm/values.yaml

```diff
@@ -1,5 +1,7 @@
 longhorn:
   nameOverride: k8s-longhorn
+  privateRegistry:
+    registrySecret: ces-container-registries
```

Comment:
Is there no way to configure multiple secrets? Not that we need it, but still...